schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,707 @@
+            "description": "The abbreviated name of the project.",
+            "type": "string",
+            "minLength": 1
+          },
+          "grant_number": {
+            "title": "Grant number",
+            "type": "string",
+            "minLength": 1
+          }
+        },
+        "additionalProperties": false
+      },
+      "language": {
+        "description": "The 2-letter ISO 639-1 language classification.",
+        "type": "string",
+        "$comment": "TODO: need complete list of language codes",
+        "enum": ["en", "de", "fr", "ru", "cn"]
+      },
+      "active_licenses": {
+        "title": "Active licenses",
+        "description": "Licenses that are selectable upon submission to arXiv.",
+        "type": "string",
+        "enum": [
+          "http://creativecommons.org/licenses/by/4.0/",
+          "http://creativecommons.org/licenses/by-sa/4.0/",
+          "http://creativecommons.org/licenses/by-nc-sa/4.0/",
+          "http://creativecommons.org/licenses/by-nc-nd/4.0/",
+          "http://arxiv.org/licenses/nonexclusive-distrib/1.0/",
+          "http://creativecommons.org/publicdomain/zero/1.0/"
+        ]
+
+      },
+      "inactive_licenses": {
+        "title": "Inactive licenses",
+        "description": "Historical licenses that are no longer selectable upon submission to arXiv.",
+        "type": "string",
+        "enum": [
+          "http://arxiv.org/licenses/assumed-1991-2003/",
+          "http://creativecommons.org/licenses/by/3.0/",
+          "http://creativecommons.org/licenses/by-nc-sa/3.0/",
+          "http://creativecommons.org/licenses/publicdomain"
+        ]
+      },
+      "person": {
+        "type": "object",
+        "properties": {
+          "full_name": {
+            "description": "The fullest representation of the person's name available in arXiv metadata.",
+            "type": "string",
+            "minLength": 1
+          },
+          "last_name": {
+            "description": "The family or surname part of the person's name, if available.",
+            "type": "string",
+            "minLength": 1
+          },
+          "first_name": {
+            "description": "The personal or forename part of the person's name, if available.",
+            "type": "string",
+            "minLength": 1
+          },
+          "suffix": {
+            "description": "The suffix part of the person's name, if available.",
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+              "Jr.",
+              "Sr.",
+              "I",
+              "II",
+              "III"
+            ]
+          },
+          "affiliations": {
+            "title": "List of affiliations",
+            "description": "Institutional affiliations",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/affiliation"
+            },
+            "minItems": 1
+          },
+          "orcid": {
```

Comment:
I remember having some conversation recently about a flag to indicate whether the ORCID is authenticated, but can't remember whether it was in the context of the WG or an AAHEP discussion.